backend/corpora/api_server/logger.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import time
+from logging import LogRecord
+from logging.config import dictConfig
+from typing import List
+
+from pythonjsonlogger import jsonlogger
+from pythonjsonlogger.jsonlogger import merge_record_extra
+
+
+# The fields to log using the json logger.
+LOGGED_FIELDS = ["levelname", "asctime", "name", "message"]
+LOG_FORMAT = " ".join([f"%({field})" for field in LOGGED_FIELDS])
+
+
+class JsonFormatter(jsonlogger.JsonFormatter):
+    default_time_format = "%Y-%m-%dT%H:%M:%S"
+    default_msec_format = "%s.%03dZ"
+
+    converter = time.gmtime
+
+    def add_required_fields(self, fields: List[str]) -> None:
```

Comment:
It's not immediately clear to the reader how these methods are being called. I assume they are hooks called by `pythonjsonlogger`. Maybe a comment to explain?